### DISCUSSION TITLE
my project's storage size seems to be wrong

### DISCUSSION THREAD

**Discussion by jayandersonsoares** [supabase/supabase#16968] [Questions] [answered]
My project's storage size seems to be wrong. My database is less than 500mb as the storage size is at 681.99 TB of 1 GB. I didn't understand how the calculation is done, since I have about 50 images with a maximum upload limit of 5mb per photo. Could someone help me understand?
![Captura de tela 2023-08-29 101236](https://github.com/supabase/supabase/assets/9561276/e46b72f9-82bf-42c0-923c-c91e9bc31047)

**Comment by GaryAustin1** [comment, marked as answer]
This is confirmed by support to be a presentation issue and not impacting your actual service.

**Reply by jayandersonsoares** [reply]
Thanks

**Reply by kevcodez** [reply]
We've rolled out a fix for storage size and this should be back to normal now.
